backends/test/suite/runner.py

```diff
@@ -142,12 +159,15 @@ def build_result(
             tester.run_method_and_compare_outputs(
                 inputs=None if generate_random_test_inputs else inputs,
                 statistics_callback=lambda stats: error_statistics.append(stats),
+                atol=1e-1,
```

Comment:
atol seems pretty high for a general default, no?

Reply:
Discussed offline. We will re-evaluate this in the future. I've created https://github.com/pytorch/executorch/issues/13347 as a backlog task to track this.